apps/emqx/src/emqx_schema.erl

```diff
@@ -523,7 +528,18 @@ fields("mqtt") ->
                 number(),
                 #{
                     default => 0.75,
-                    desc => ?DESC(mqtt_keepalive_backoff)
```

Comment:
nit: When we deprecate/hide this field, is it necessary to delete its i18n description? Should we keep it?

Reply:
delete